addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_sampler_keyframes.py

```diff
@@ -116,13 +116,13 @@ def gather_keyframes(blender_object_if_armature: typing.Optional[bpy.types.Objec
             pose_bone_if_armature = None
 
         # sample all frames
-        time = start
-        step = 1.0 / bpy.context.scene.render.fps
-        while time <= end:
-            key = Keyframe(channels, time)
+        frame = start_frame
+        step = export_settings['gltf_frame_step']
```

Comment:
Maybe we should drop this parameter, to avoid clutter in the export settings?

Reply:
How would user then specify step value? Depending on your animation adjusting the value to control quality vs. file size is very useful IMO :)

Reply:
I consider it a useful setting, as baked animation can otherwise be quite large. If there's no objection other than UI clutter I'd prefer to keep it.